### DISCUSSION TITLE
Where to execute Javascript to obtain cookie?

### DISCUSSION THREAD

**Discussion by SuperDarke14** [ltworf/localslackirc#410] [General]
Hi there,

I am trying to set up my own local server to access my Slack workplace. I have obtained my token but I have no clue where to execute the provided javascript to obtain a cookie. Could anyone give me a hand here?

Thanks.

**Comment by ltworf** [comment]
You need to open slack in your browser, in the developer tools open the network monitor.

Then do some traffic in slack (like refresh the page or something). From the requests that happen, you see the cookies. And then take the interesting one and give it to localslackirc.

**Comment by SuperDarke14** [comment]
Hello, 
Turns out I was being a dingus and I had written it like this:
`COOKIE= d=01234;`
instead of this:
`COOKIE=d=01234;`.
I've successfully made it work.
